### PR TITLE
Search AG_Handout_kits for kit_id and barcode

### LINKED ISSUE #641: Add search for ag_handout_kits table
If we issue a handout kit,  the only way to check the credentials is to use sql developer.   It would be nice to have a wwwadmin page to search this so if someone emails the help account we can get them the information they need without using sql developer.

### PR DESCRIPTION
**Pull request by teravest**
Now that we are doing handout kits if a participant losses credentials sql is currently the only way to retrieve them. This change addes ag_handout_kits to the search page.  
If a kit_id or barcode is not found in the normal tables than ag_handout_kits is searched. 

barcodes to test with 
000000001 - normal results
000015537 - ag_handout_kit results with one barcode in the kit
000015522 - ag_handout_kit with 5 barcodes in the kit
010000000 - no results found

fixes #641

### CODE REVIEW

File: www/wwwadmin/ag_search.psp [hunk 1]
```diff
@@ -260,6 +268,28 @@ if 'search_term' in form:
 				# Close the div for this login
 				req.write('</div>')
 				
+	elif len(kit_list) > 0:
```
Comment (adamrp) [inline www/wwwadmin/ag_search.psp]: This block only executes if `len(login_list)` is greater than 0; is that the intent?  It seems to me that you would want to display this information regardless.

Comment (teravest) [inline www/wwwadmin/ag_search.psp]: actually it only executes if len(login_list) is <= 0.  The information is
already displayed if the barcode has a log in associated with it.  Its the
case where it doesn't that we need to get the info from ag_handout_kits.

Emily

**Comment by adamrp** [comment]
Woops, that's what I meant -- wrong inequality!  I see now, though.  Could you change the name of the variable from `kit_list` to `handout_kit_list` to eliminate the ambiguity?

File: www/wwwadmin/ag_search.psp [hunk 2]
```diff
@@ -260,6 +268,28 @@ if 'search_term' in form:
...
+			where kit_id = '{0}'""".format(kit_id)
+			results = ag_data_access.dynamicMetadataSelect(sql).fetchall()
+			if len(results) > 0:
+				req.write('<table class="widetable">')
+				req.write('<th>kit_id</th>')
+				req.write('<th>password</th>')
+				req.write('<th>barcode</th>')
+				req.write('<th>verification_code</th>')
+				for row in results:
+						tableout= '</td><td>'.join(row)
```
Comment (ElDeveloper) [inline www/wwwadmin/ag_search.psp]: There seem to be a few extra tabs in this blocks and adding a comment explaining the the row creation would be awesome.

**Comment by ElDeveloper** [comment]
t0klc0c

**Pull request merged**